src/main/java/uk/gov/justice/digital/client/dynamodb/DynamoDBClientProvider.java

```diff
@@ -6,25 +6,25 @@
 import jakarta.inject.Inject;
 import jakarta.inject.Singleton;
 import uk.gov.justice.digital.client.ClientProvider;
-import uk.gov.justice.digital.config.JobParameters;
+import uk.gov.justice.digital.config.JobArguments;
 
 
 @Singleton
 public class DynamoDBClientProvider implements ClientProvider<AmazonDynamoDB> {
 
-    private final JobParameters jobParameters;
+    private final JobArguments jobArguments;
 
     @Inject
-    public DynamoDBClientProvider(JobParameters jobParameters) {
-        this.jobParameters = jobParameters;
+    public DynamoDBClientProvider(JobArguments jobArguments) {
+        this.jobArguments = jobArguments;
     }
 
     @Override
     public AmazonDynamoDB getClient() {
         return AmazonDynamoDBClientBuilder.standard()
                 .withEndpointConfiguration(new AwsClientBuilder.EndpointConfiguration(
-                        jobParameters.getAwsDynamoDBEndpointUrl(),
```

Comment:
Can we make "AWS" upper case on these method names consistent with "DB" string?

Reply:
This doesn't follow java naming conventions so will be left as is.